netns/netns.go

```diff
@@ -0,0 +1,28 @@
+package netns
+
+import "github.com/vishvananda/netns"
+
+type Netns struct{}
+
+func NewNetns() *Netns {
```

Comment:
```suggestion
func New() *Netns {
```
you will always call this through the `netns` package, so it is unambiguous to say `netns.New()`